docs/guides/configure-error-suppression.mdx

```diff
@@ -1,13 +1,15 @@
 ---
-title: Configure runtime compilation
-description: How to configure runtime compilation in Qiskit Runtime.
+title: Configure error suppression
+description: How to configure error suppression in Qiskit Runtime. (Previous topic - runtime compilation)
 
 ---
 
-# Configure runtime compilation for Qiskit Runtime
+# Configure error suppression for Qiskit Runtime
 
 Runtime compilation techniques transform your circuit to minimize errors. Runtime compilation adds some classical pre-processing overhead to your overall runtime. Therefore, it is important to achieve a balance between perfecting your results and ensuring that your job completes in a reasonable amount of time.
```

Comment:
Since we changed the topic to "error suppression", we should talk about error suppression instead of compilation (compilation is just the tool we use to apply error suppression instructions). 

Stealing words from this [article](https://www.ibm.com/quantum/blog/quantum-error-suppression-mitigation-correction), we can have an intro similar to that of error mitigation page:

"Error suppression refers to techniques where you use knowledge about the undesirable effects to introduce customization that can anticipate and avoid the potential impacts of those effects. These techniques often consist of altering or adding control signals to ensure that the quantum processor returns the desired results.

V2 primitives support a number of error suppression techniques, including dynamical decoupling and Pauli twirling. See [Error mitigation and suppression techniques](https://docs.quantum.ibm.com/guides/error-mitigation-and-suppression-techniques) for explanation of each. In V2 primitives, you can turn on or off individual method. See the [Advanced error suppression options] section for details."